trento/xml/article_sap_trento.xml

```diff
@@ -2272,29 +2272,33 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
 
   <section xml:id="sec-trento-updating-trentoserver">
     <title>Updating &t.server;</title>
-    <para> The procedure to update the &t.server; depends on the deployment type: &k8s;, systemd or containerized.
-    The current content of this section is specific for &kube; deployments.</para>
-   <para>
-     If &t.server; was installed manually, then it must be updated manually using the latest versions
-     of the container images available in the SUSE public registry. If it was installed using
-     Helm chart, it can be updated using the same Helm command as for the installation:</para>
-   <screen>helm upgrade \
-   --install trento-server oci://registry.suse.com/trento/trento-server \
-   --set trento-web.trentoWebOrigin=<replaceable>TRENTO_SERVER_HOSTNAME</replaceable> \
-   --set trento-web.adminUser.password=<replaceable>ADMIN_PASSWORD</replaceable>
-        </screen>
-   <para>A few things to consider:</para>
+    <para> The procedure to update &t.server; depends on the chosen deployment option: &k8s;, systemd or containerized.</para>
+      <para>A few general considerations when performing an update:</para>
       <itemizedlist>
-                <listitem>
+         <listitem>
           <para>Before updating &t.server; ensure that all the &t.agent;s in the environment are supported by the target version.
            See section <xref linkend="sec-trento-compatibility-matrix" /> for details.
           </para>
         </listitem>
-        <listitem>
+       <listitem>
+        <para>When updating from an older Trento version to version 2.4 or higher, the admin password might have to
+         be adjusted to meet the rules described at the User Managmeent section .</para>
```

Comment:
If you want to refer to another section inside this article, use `<xref linkend="TARGET_ID"/>`. However, I didn't find such a section. Any hints?

Reply:
@tomschr it would be
`<xref linkend="sec-trento-user-management"/>`
where sec-trento-user-management is introduced by https://github.com/SUSE/doc-unversioned/pull/165.

Reply:
That would mean, this PR cannot be valid until #165 is merged. Ok, let's keep this open.

Reply:
Perfect @tomschr . Thank you!